source/CIGameSettings.h

```diff
@@ -124,15 +124,15 @@ class GameSettings {
     /**
      * Returns the planet mass required to win the game.
      */
-    uint16_t getPlanetMassToWin() const {
-        return _planetMassToWin;
+    uint16_t getPlanetDustPerLayer() const {
```

Comment:
NIT: can you rename this to _planetStardustPerLayer

Reply:
Changed 👍